src/views/conference/index/index.jsx

```diff
@@ -21,10 +21,17 @@ var ConferenceSplash = React.createClass({
                     <h3>
                         Scratch Conference 2016 | Cambridge, MA, USA
                     </h3>
+                    <h3>
+                        August 4 - 6
```

Comment:
Let's move this up and take out 'Scratch Conference' so it reads:

'August 4–6, 2016 | Cambridge, MA, USA'